### PR TITLE
created FirstBallIntoTheShooter and VerticalConveyorTimedCommand

### PR DESCRIPTION
**Pull request by JosephJackson**
Justina and Ariella's new FirstBallIntoTheShooter auto command and supporting command

### CODE REVIEW

File: y2022/RapidReact/src/main/java/com/gos/rapidreact/auto_modes/FirstBallIntoTheShooter.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import com.gos.rapidreact.commands.ShooterRpmPIDCommand;
+import com.gos.rapidreact.commands.VerticalConveyorTimedCommand;
+import com.gos.rapidreact.subsystems.ChassisSubsystem;
+import com.gos.rapidreact.subsystems.ShooterSubsystem;
+import com.gos.rapidreact.subsystems.VerticalConveyorSubsystem;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class FirstBallIntoTheShooter extends SequentialCommandGroup {
+    public FirstBallIntoTheShooter(ChassisSubsystem chassis, ShooterSubsystem shooter, VerticalConveyorSubsystem verticalConveyor) {
+        super(new DriveDistanceCommand(chassis, .762, 0.05),
```
Comment (pjreiniger) [inline y2022/RapidReact/src/main/java/com/gos/rapidreact/auto_modes/FirstBallIntoTheShooter.java:14]: I'm a dumb american, can you do this in inches and do `Units.inchesToMeters()` . The number should probably be a constant as well, probably OK jsut to have in this class

File: y2022/RapidReact/src/main/java/com/gos/rapidreact/auto_modes/FirstBallIntoTheShooter.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            new ShooterRpmPIDCommand(shooter, 2500),
```
Comment (pjreiniger) [inline y2022/RapidReact/src/main/java/com/gos/rapidreact/auto_modes/FirstBallIntoTheShooter.java:15]: Make a constant, either in `ShooterSubystem` or `Constants`

File: y2022/RapidReact/src/main/java/com/gos/rapidreact/auto_modes/FirstBallIntoTheShooter.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            new VerticalConveyorTimedCommand(verticalConveyor, 10));
```
Comment (pjreiniger) [inline y2022/RapidReact/src/main/java/com/gos/rapidreact/auto_modes/FirstBallIntoTheShooter.java:16]: Make a constant, either in `VerticalConveyorSubsystem` or `Constants`

**Review by pjreiniger — CHANGES_REQUESTED** [review]
Functionally this all makes sense and looks good. Just a little bit of cleanup requested.

File: y2022/RapidReact/src/main/java/com/gos/rapidreact/auto_modes/FirstBallIntoTheShooter.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+import com.gos.rapidreact.commands.DriveDistanceCommand;
+import com.gos.rapidreact.commands.ShooterRpmPIDCommand;
+import com.gos.rapidreact.commands.VerticalConveyorTimedCommand;
+import com.gos.rapidreact.subsystems.ChassisSubsystem;
+import com.gos.rapidreact.subsystems.ShooterSubsystem;
+import com.gos.rapidreact.subsystems.VerticalConveyorSubsystem;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class FirstBallIntoTheShooter extends SequentialCommandGroup {
```
Comment (ashley-wei) [inline y2022/RapidReact/src/main/java/com/gos/rapidreact/auto_modes/FirstBallIntoTheShooter.java:12]: change name of class --> not putting it into the shooter (into the hub) & doesn't specify auto. also helpful to specify that this is the ball in the cargo or just call it something like 'one ball auto'

File: y2022/RapidReact/src/main/java/com/gos/rapidreact/commands/VerticalConveyorTimedCommand.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
+package com.gos.rapidreact.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import com.gos.rapidreact.subsystems.VerticalConveyorSubsystem;
+import edu.wpi.first.wpilibj.Timer;
+
+
+public class VerticalConveyorTimedCommand extends CommandBase {
```
Comment (pjreiniger) [inline y2022/RapidReact/src/main/java/com/gos/rapidreact/commands/VerticalConveyorTimedCommand.java:8]: [Last year ](https://github.com/GirlsOfSteelRobotics/GirlsOfSteelFRC/blob/main/y2020/2020InfiniteRecharge/src/main/java/com/gos/infinite_recharge/auto_modes/ShootAndDriveToTrench.java#L52) we used [decoraters](https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html?highlight=withtimeout) for stuff like timeouts

**Review by pjreiniger — APPROVED** [review]

**Pull request merged**